zap/src/main/java/org/parosproxy/paros/extension/ExtensionLoader.java

```diff
@@ -862,7 +864,8 @@ private void hookAllExtension(double progressFactor) {
         for (int i = 0; i < getExtensionCount(); i++) {
             final Extension ext = getExtension(i);
             try {
-                logger.info("Initializing " + ext.getDescription());
+                String name = ext.getUIName() == null ? ext.getName() : ext.getUIName() + " - ";
+                logger.info("Initializing {} {}", name, ext.getDescription());
```

Comment:
The dash could go in the log message instead of being appended to `name`.
```suggestion
                String name = ext.getUIName() == null ? ext.getName() : ext.getUIName();
                logger.info("Initializing {} - {}", name, ext.getDescription());
```